src/stores/Binding/Store.ts

```diff
@@ -1024,8 +1021,18 @@ const getInitialState = (
                     defaultInterval &&
                     POSTGRES_INTERVAL_RE.test(defaultInterval)
                 ) {
-                    state.defaultCaptureInterval =
+                    const intervalObject =
                         Duration.fromISOTime(defaultInterval).toObject();
+
+                    Object.entries(intervalObject).forEach(
+                        ([timeUnit, unitValue]) => {
+                            if (unitValue === 0) {
+                                intervalObject[timeUnit] = undefined;
+                            }
+                        }
+                    );
```

Comment:
These should be filtered out in the format function and not the store. I know this means we'll run the filter more but it isn't that huge of a piece of logic.

Reply:
The format function does not alter the default interval, but I can look into rearranging things to make that work.

Reply:
Just make it a boolean option so we can control it just for the default output.

The goal is to have this logic where it should be eventually when luxon does add this feature : https://github.com/moment/luxon/issues/1134

Reply:
We could also just make a new function just for the defaulted option... since it is a special case and could be easier to understand than an extra boolean.